alonzo/formal-spec/utxo.tex

```diff
@@ -5,8 +5,8 @@ \section{UTxO}
   \emph{Functions}
   %
   \begin{align*}
-    & \fun{isNonNativeScriptAddress} \in \Tx \to \Addr \to \Bool \\
-    & \fun{isNonNativeScriptAddress}~tx~a = \\
+    & \fun{isTwoPhaseScriptAddress} \in \Tx \to \Addr \to \Bool \\
+    & \fun{isTwoPhaseScriptAddress}~tx~a = \\
```

Comment:
this is certainly an improvement, I appreciate it, now the code and the spec agree. unfortunately :grimacing: we may have to change it again, just a heads up, sorry